__tests__/app.test.js

```diff
@@ -4,6 +4,7 @@ const seed = require('../db/seeds/seed');
 const testData = require('../db/data/test-data/');
 const connection = require('../db/connection');
 const db = require("../db/connection");
+//import { sorted } from 'jest-sorted';
```

Comment:
looks like an auto import from VSCode; can be destroyed. Aim to avoid committing commented out code